newrelic-agent/src/main/java/com/newrelic/agent/config/SystemPropertyProvider.java

```diff
@@ -128,21 +110,6 @@ private void addPropertyWithoutSystemPropRoot(Map<String, Object> nrProps, Strin
     }
 
     public String getEnvironmentVariable(String key) {
-        String propVal = envVars.get(key);
```

Comment:
This entry consisted of exactly one value: `envVars.put(LOG_FILE_NAME, getenv(LOG_ENV));`. I opted to remove this special case from here, and pull it up into `AgentConfigImpl`.